tests/ecosystem/deployment/test_deployment.py

```diff
@@ -15,9 +16,139 @@ def test_deployment():
     deploy = config.RUN['cli_params'].get('deploy')
     teardown = config.RUN['cli_params'].get('teardown')
     if not teardown or deploy:
-        assert is_cluster_running(config.ENV_DATA['cluster_path'])
+        ocs_install_verification()
 
     if teardown:
         log.info(
             "Cluster will be destroyed during teardown part of this test."
         )
+
+
+def ocs_install_verification():
+    """
+    Perform steps necessary to verify a successful OCS installation
+    """
+    namespace = config.ENV_DATA['cluster_namespace']
+    storage_cluster_name = config.ENV_DATA['storage_cluster_name']
+
+    # Verify cluster is running
+    assert is_cluster_running(config.ENV_DATA['cluster_path'])
+
+    # Verify OCS Operator and Local Storage Operator in Succeeded phase
+    csv = ocp.OCP(kind='csv', namespace=namespace)
+    csvs = csv.get()
+    for item in csvs['items']:
+        name = item['metadata']['name']
+        log.info("Checking status of %s", name)
+        assert item['status']['phase'] == 'Succeeded', (
+            f"Operator {name} not 'Succeeded'"
+        )
+
+    # Verify OCS Cluster Service (ocs-storagecluster) is Ready
+    storage_cluster = ocp.OCP(kind='StorageCluster', namespace=namespace)
+    storage_clusters = storage_cluster.get()
+    for item in storage_clusters['items']:
+        name = item['metadata']['name']
+        log.info("Checking status of %s", name)
+        assert item['status']['phase'] == 'Ready', (
+            f"StorageCluster {name} not 'Ready'"
+        )
+
+    # Verify pods in running state and proper counts
+    pod = ocp.OCP(
+        kind=constants.POD, namespace=namespace
+    )
+    timeout = 0
+    # ocs-operator
+    assert pod.wait_for_resource(
+        condition=constants.STATUS_RUNNING,
+        selector='name=ocs-operator',
+        timeout=timeout
+    )
+    # storagecluster-operator
+    assert pod.wait_for_resource(
+        condition=constants.STATUS_RUNNING,
+        selector=f'app={storage_cluster_name}-operator',
```

Comment:
I think the pod names are still 'rook-ceph' and independent of `storage_cluster_name`

Reply:
Is that for this specific pod or all of the pods? @vasukulkarni

Reply:
@clacroix12 In this PR's jenkins job, I see all the pods are still prefixed with `rook-ceph`, I did change the cluster_name https://github.com/red-hat-storage/ocs-ci/pull/920

Reply:
Fixed. We're now reading in the pod selector from `constants`.